contracts/src/Summa.sol

```diff
@@ -16,111 +16,50 @@ contract Summa is Ownable {
         bytes32 mstRoot;
         uint timestamp;
         uint ethBlockNumber;
-        bytes32 btcBlockHash;
-        address signingPubKey;
-        uint commitPeriodEnd;
     }
 
     mapping(uint => RoundInformation) public rounds;
-
-    mapping(uint => bytes32[]) public userAccountCommitments;
-
-    //By round index and accountDataHash
-    mapping(uint => mapping(bytes32 => bool))
-        public accountDataCommitmentStatus;
+    mapping(uint => RoundInformation) public proofs;
 
     struct PublicInput {
```

Comment:
Just quick comment (not part as a complete review). 

- As discussed in the meeting today, for now we can remove the necessity to work with oracles or axiom-like service just by querying the balance of the CEX at the current block. This makes `ethBlockNumber` no longer relevant inside the `PublicInput` struct
- The `signatures` and the `cexETHPubKey` and the `round` are not public input of the circuits (as there's no check performed on them inside the circuit) but are sent to the smart contract by the CEX **together** with the proof. Is that ok for you? also from a security standpoint.
- The public input of the circuit is now exactly `cexETHBalances[]` but, rather, the `cexETHSum`. I guess that in the smart contract we should check that `cexETHBalances[] >= cexETHSum`